jellyfin_client.py

```diff
@@ -20,6 +20,7 @@ def _post(self, endpoint, payload: Optional[dict] = {}) -> bool:
         payload['api_key'] = self.api_key
         r = self.session.post(
             url='{}/{}'.format(self.url, endpoint), params=payload)
+        return r.status_code == 200
```

Comment:
Since you might have a testsetup still running, could you maybe add a catch around this as well?
This (and how it was also) masks non 2xx responses from the server which isn't great:

https://github.com/wilmardo/migrate-plex-to-jellyfin/issues/33#issuecomment-3455559622

Just throw an error with the requests body, it doesn't have to be that pretty :)